### DISCUSSION TITLE
[Enhancements] Various Difficulty Options

### DISCUSSION THREAD

**Discussion by MrEveryDay98** [HarbourMasters/2ship2harkinian#530] [Ideas]
- Trigger the "Terrible Fate" cutscene/time reset upon death
- Ability to scale various types of damage (fall damage, void damage, explosions, blades, monsters, etc.)
- Ability to make void deaths instantly trigger the Game Over event without teleporting Link back to an entrance
- Scalable monster speed

**Comment by smashbro596** [comment]
maybe we could also have sliders for our weapons, to make them as strong or as weak as we need to.
